data/companies.json

```diff
@@ -721,6 +721,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "Vertex Web Surf Pvt Ltd.",
+      "establishedAt": 1453832100,
+      "location": "Samakhusi, Kathmandu",
+      "website": "https://www.vertexwebsurf.com.np/",
+      "email": "info@vertexwebsurf.com.np",
+      "phone": "01-4388511",
```

Comment:
The format for the phone number is `+977-14388511`

Reply:
> The format for the phone number is `+977-14388511`

I checked but the some previously merged codes don't follow this formatting for phone number. So, I let it be.